packages/gnome_settings_daemon.rb

```diff
@@ -0,0 +1,74 @@
+# Deprecated: Newer versions are no longer compatible with Chromebrew due to dependence on systemd.
+
+require 'package'
+
+class Gnome_settings_daemon < Package
```

Comment:
Use `< Meson` instead of `< Package`?  This will require `require 'buildsystems/meson'` above as well.